### PR TITLE
fix(skills): issue in deletion of topics in skills

### PR DESCRIPTION
**Pull request by priyajeet**
Fixes an issue with skills topics card where the order of deletion had an incorrect sort. Also refactors a bit for better testability.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
         "build:ci:es": "yarn build:prod:es",
         "build:ci:dist": "LANGUAGE=en-US REACT=true yarn build:prod:dist",
         "build:dev:dist": "LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev webpack --config conf/webpack.config.js --mode development --progress --colors",
-        "build:dev:es": "BABEL_ENV=dev NODE_ENV=dev yarn build:es --copy-files --source-maps inline --watch --ignore \"**/__tests__/**\",\"src/components/messages.js\"",
+        "build:dev:es": "BABEL_ENV=dev NODE_ENV=dev yarn build:es --copy-files --source-maps inline --watch --ignore \"**/__tests__/**\"",
```
Comment (jstoffan) [inline package.json]: Can you move this to a separate PR?
